index.js

```diff
@@ -3,16 +3,49 @@ const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
+
 /***** Deliverable 2 *****/
 
+let headerColor = document.getElementById('header')
+headerColor.style.color = 'blue';
+
+
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+let foundDiv = document.querySelector('.traveler')
+    foundDiv.innerHTML +=
+    `<div id="profile">
+      <img src="${traveler.photo}"
+        alt="${traveler.name}">
+      <h2>${traveler.name}</h2>
+      <em>${traveler.nickname}</em>
+    </div>`
+
+
+console.log('This is what the traveler object looks like: ', traveler)
+
+
 
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function (postObj){
+    foundDiv.innerHTML += 
+`<li data-id="${postObj.id}">
+  <p>${postObj.description}</p>
+  <img src="${postObj.photo}" alt="${postObj.species}"/>
+  <a href="${postObj.link}" target="_blank">Here's a video about the ${postObj.species} species!</a>
+</li>`
```

Comment:
This is a situation where `+=` will work! However, you'd want to use the `ul#animals` as the parent element for each of the `<li>` elements, not the `div.traveler`.

One word of caution: Just be careful of using `innerHTML +=` since it _destroys_ any existing DOM elements inside the element. This can cause have unintended consequences when we start adding event listeners to DOM elements.